.github/workflows/CI.yml

```diff
@@ -77,3 +77,22 @@ jobs:
           TOKEN='ghp_i5wtj1l2AbpFv3OU96w6R'
           TOKEN+='On3bHOkcV2AmVY6'
           DANGER_GITHUB_API_TOKEN=$TOKEN pnpm danger ci || $( exit 0 )
+
+  scripts:
```

Comment:
does this happen for every CI run?

Reply:
Yes. I can make it only happen on pushes to master or PRs that change the files, I guess, but funnily the scripts depend on other type packages so they may break our scripts (which is a good test case).

Reply:
Sounds fine then.